core/handlers/msg_created.go

```diff
@@ -15,45 +15,9 @@ import (
 )
 
 func init() {
-	models.RegisterEventPreWriteHandler(events.TypeMsgCreated, handlePreMsgCreated)
 	models.RegisterEventHandler(events.TypeMsgCreated, handleMsgCreated)
 }
 
-// handlePreMsgCreated clears our timeout on our session so that courier can send it when the message is sent, that will be set by courier when sent
-func handlePreMsgCreated(ctx context.Context, rt *runtime.Runtime, tx *sqlx.Tx, oa *models.OrgAssets, scene *models.Scene, e flows.Event) error {
```

Comment:
this is now done where we create/update the session objects.. and it was the only "EventPreWriteHandler" so those are no longer a thing